src/openApi/openApiWrapper.ts

```diff
@@ -136,4 +137,12 @@ export default class OpenApiWrapper {
     this.correlationId = existingCorrelationId ?? uuid.v4();
     return this.correlationId;
   }
+
+  private redactObjectsSecret(obj: any) {
+    const err = obj;
+    if (err.config?.data !== undefined) {
+      err.config.data = redactSecret(err.config.data);
```

Comment:
Here we are changing the `data` to `string`. I'm not sure if calling `serializeObject` on the `error` fixes this, but perhaps we should call `serializeObject` on the result of `redactSecret` here